tests/integrations/rediscluster/test_rediscluster.py

```diff
@@ -0,0 +1,51 @@
+from sentry_sdk import capture_message
+import sentry_sdk.integrations
+from sentry_sdk.integrations.redis import RedisIntegration
+
+import rediscluster
+
+
+def _test_rediscluster_basic(sentry_init, capture_events, monkeypatch, clsname):
+
+    redis_cluster_cls = getattr(rediscluster, clsname)
+
+    execute_command_calls = []
+
+    def execute_command(*args, **kwargs):
+        execute_command_calls.append((args, kwargs))
+
+    monkeypatch.setattr(
+        redis_cluster_cls, "execute_command", execute_command,
+    )
+
+    # should be done by the sentry_init fixture?
+    sentry_sdk.integrations._installed_integrations = set()
```

Comment:
Why do we need to do this?

Reply:
I need the `RedisIntegration.setup_once` to run for both tests.